packages/web/pages/_document.tsx

```diff
@@ -24,6 +24,8 @@ class MetaDocument extends Document {
             rel="stylesheet"
           />
           <link rel="shortcut icon" href="/favicon.png" />
+          {/* @TODO: remove and apply filter with _hover={} when https://github.com/chakra-ui/chakra-ui/issues/2831 is resolved */}
```

Comment:
They said this is fixed in v1.x of chakra UI, did you try updating it ?

Reply:
I'm going to have to roll in updates using the migration guide.